interview-questions.md

```diff
@@ -8,50 +8,50 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is the difference between a parameter and an argument?
 
-Your answer:
+Your answer: Parameters are like names and arguements are the values passed in. So for example a parameter would be something like (name) but the arguement, the actual value would be my name, "Zeke".
 
-Researched answer:
+Researched answer: A parameter is a variable we name that is then passed into a function. They are used to pass arguements into a function. The arguement has the actual values in it. Arguements are used to get to the parameter. 
 
 2. The JavaScript built in method .map() takes predefined parameters. What are they? Which are required and which are optional?
 
-Your answer:
+Your answer: .map is told to iterate through the whole array or string. It can also be told to manipulate each index for example multiply all the numbers by 2, and it will return all the numbers back multiplied by 2. 
 
-Researched answer:
+Researched answer: .map will create a new array and it calls each element once in an array. 
```

Comment:
Make sure that you are answering the same question that is being asked.  In this case, .map takes currentValue, index, and array as parameters but only currentValue is required.